src/main/java/reposense/parser/RepoConfigCsvParser.java

```diff
@@ -9,6 +9,7 @@
 public class RepoConfigCsvParser extends CsvParser<RepoConfiguration> {
     public static final String REPO_CONFIG_FILENAME = "repo-config.csv";
     private static final String IGNORE_STANDALONE_CONFIG_KEYWORD = "yes";
+    private static final String DEFAULT_BRANCH = "HEAD";
```

Comment:
Can just create this constant in `CsvParser` so that you don't have to initialize it in both child Parser.